File: 4_Sorting_Algorithms/Insertion_Sort/Python/main.py
"""
This is my main module
"""
from insertion_sort import insertion_sort


def main():
    arr = [64, 34, 25, 12, 22, 11, 90]
    print("Original array: {}".format(arr))
    insertion_sort(arr)
    print("Sorted array: {}".format(arr))
    print('The array is sorted correctly.') if sorted(arr) == arr else print("The array is not sorted correctly.")


if __name__ == '__main__':
    main()

File: 4_Sorting_Algorithms/Insertion_Sort/C/main.c
#include "insertion_sort.h"

/**
 * main - This is my main function
 * 
 * Return: 1
 */
int main(void){
    int arr[] = {64, 34, 25, 12, 22, 11, 90};
    int arr_size = sizeof(arr)/sizeof(arr[0]);

    printf("Original array:");
    display_array(arr, arr_size);

    insertion_sort(arr , arr_size);

    printf("Sorted array:");
    display_array(arr, arr_size);
    is_sorted(arr, arr_size) ? printf("The array is sorted correctly.\n"): printf("The array is not sorted correctly.\n");

    return 0;
}